scripts/lighthouse/runLighthouse.js

```diff
@@ -0,0 +1,58 @@
+const lighthouse = require('lighthouse'); // eslint-disable-line import/no-extraneous-dependencies
+const chromeLauncher = require('chrome-launcher'); // eslint-disable-line import/no-extraneous-dependencies
+
+let config = {};
+
+function launchChromeAndRunLighthouse(url, opts, lConfig = null) {
+  const options = opts;
```

Comment:
Any reason why we don't name the parameter itself `options`?

Reply:
I am not sure what clarity that would add.  
`opts` needs to be assigned to a const or it will fail the `no-param-reassign` rule